src/Service/ValueService.php

```diff
@@ -188,16 +188,17 @@ public function connectSubObjects(Value $value): void
 
             $value->setArrayValue([]);
             $value->setStringValue(null);
-        } else if ((Uuid::isValid($value->getStringValue()) === true || filter_var($value->getStringValue(), FILTER_VALIDATE_URL) === true) && $identifier = $value->getStringValue()) {
+        } else if ((Uuid::isValid($value->getStringValue()) === true || filter_var($value->getStringValue(), FILTER_VALIDATE_URL) !== false) && empty($value->getStringValue()) === false) {
```

Comment:
Why change the === true to  !== false?

Reply:
Because === true is incorrect here and causes subobjects to not be synced at all for urls
![image](https://github.com/CommonGateway/CoreBundle/assets/3785488/469edaf1-b51e-40d5-adb7-2a9ec17fe95b)